librz/util/str.c

```diff
@@ -756,19 +756,11 @@ RZ_API char *rz_str_newf(const char *fmt, ...) {
 // Secure string copy with null terminator (like strlcpy or strscpy but ours
 RZ_API size_t rz_str_ncpy(char *dst, const char *src, size_t n) {
 	rz_return_val_if_fail(dst && src, 0);
-	size_t i;
 
-	// do not do anything if n is 0
```

Comment:
I think it's better to still keep the `0` check.

Reply:
I'll try bringing back the same check, then.